lib/arjdbc/vertica/column.rb

```diff
@@ -17,7 +17,7 @@ def self.cast_type_from_data_type_id(data_type_id)
         when 5 #, 'bool', :bool
           ::ActiveRecord::Type::Boolean.new
         when 6 #, 'integer', :integer
-          ::ActiveRecord::Type::Integer.new
+          ::ActiveRecord::Type::BigInteger.new
```

Comment:
>Vertica does not support the SQL/JDBC types NUMERIC, SMALLINT, or TINYINT.

https://www.vertica.com/docs/9.2.x/HTML/Content/Authoring/SQLReferenceManual/DataTypes/Numeric/INTEGER.htm

This appears to be a :+1: change.